website/package.json

```diff
@@ -14,6 +16,9 @@
   },
   "dependencies": {
     "@aragon/ui": "^1.0.0",
+    "@aragon/wrapper": "^5.0.0-rc.26",
+    "clipboard-polyfill": "^2.8.6",
+    "crypto-js": "^4.0.0",
```

Comment:
What are these used for?

Reply:
`clipboard-polyfill` is used by the `AccountModule` for copying the address to the clipboard; the `crypto-js` library is for being able to take the `isAddress()` [implementation](https://github.com/Evalir/apiary/blob/master/website/src/utils/web3-utils.js#L64) from `web3`: